### ISSUE TITLE
No error if inspection query text is emty (f.e. SQL file is not available)

### ISSUE THREAD

**Issue by lospejos** [idlesign/pg_analyse#5]
There is no error if inspection SQL file (or all of them) is not available, because it seems that empty text is a valid SQL statement.
I believe it's not what user expects from this tool.
At least `pg_analyse` could check if SQL text to execute is empty (f.e. empty correspondent file, file is absent, file is not accessible due to permissions issues etc) and print some warning about it rather than just report that no issues found for certain inspection, because actually inspection was not executed.

F.e.:
```
Tables without Primary Key [tbl_nopk]
Warning: inspection query text is empty, results may be inaccurate!
  Table name    Table size
  ------------  ------------
```

**Comment by lospejos** [comment]
I checked once again and it seems that it was some misconfiguration in my project, when SQL file is not available, an error appears:

`[Errno 2] No such file or directory: '<path>\pg_analyse\pg_analyse\sql\contrib\index_health\sql\indexes_with_null_values.sql'`
